datastore/redis.go

```diff
@@ -22,7 +23,7 @@ import (
 var (
 	redisPrefix = "boost-relay"
 
-	expiryBidCache = 45 * time.Second
+	expiryBidCache = 1000 * time.Second
```

Comment:
this is a massive increase. can you justify it? expiry matters a lot b/c it could dramatically change the cost of the redis instance.

Reply:
nah nah this was for testing.